clang/test/Driver/sycl-spirv-ext.c

```diff
@@ -0,0 +1,108 @@
+// REQUIRES: system-linux
```

Comment:
Can we also consider a Windows test, or modify this one to work for both Windows and Linux?